### PR TITLE
Match 6 functions, tidy up param types in Menu/

### PR DESCRIPTION
**Pull request by nocato**

### CODE REVIEW

File: src/Menu/menu_right.c [hunk 1]
```diff
@@ -199,9 +199,97 @@ int sub_8003D568()
...
+    }
+
+    *arg2 = temp_v0;
+    temp_v1 = dword_800ABAE0 * 0x28;
+    temp_lo = arg1 * temp_v1;
+    var_v0 = temp_lo >> 0x10;
+    if (temp_lo < 0)
+    {
+        temp_lo = temp_lo + 0xFFFF;
+        var_v0 = temp_lo >> 0x10;
```
Comment (KieronJ) [inline src/Menu/menu_right.c:234]: Can these + 0xffff, >> 0x10 sequences be turned into division by 65536?

File: src/Menu/menu_right.c [hunk 2]
```diff
@@ -78,7 +78,15 @@ void sub_8003CE78(void)
 #pragma INCLUDE_ASM("asm/sub_8003CEF8.s") // 232 bytes
 #pragma INCLUDE_ASM("asm/sub_8003CFE0.s") // 144 bytes
 #pragma INCLUDE_ASM("asm/sub_8003D070.s") // 96 bytes
-#pragma INCLUDE_ASM("asm/sub_8003D0D0.s") // 84 bytes
+
+void sub_8003D0D0(SPRT *pPrim, MenuMan_Inventory_14h_Unk *pUnk, int offset_x, int offset_y)
+{
+    pPrim->x0 = (signed char)pUnk->field_9_x + offset_x; // Casting to signed char to get a match
+    pPrim->y0 = (signed char)pUnk->field_A_y + offset_y;
+    *(int *)&pPrim->u0 = *(int *)&pUnk->field_C_u; // Copy field_C_u, field_D_v, field_E_clut
```
Comment (KieronJ) [inline src/Menu/menu_right.c:86]: This is likely the LCOPY macro, could you see if it will match using that?

**Review by KieronJ — CHANGES_REQUESTED** [review]

**Comment by nocato** [comment]
@KieronJ Fixed both issues (`/ 65536` and `LCOPY`)  in the forced-pushed commits.

**Review by KieronJ — APPROVED** [review]

**Comment by KieronJ** [comment]
> @KieronJ Fixed both issues (`/ 65536` and `LCOPY`) in the forced-pushed commits.

Looks great, thanks!

**Pull request merged**